frontend/src/components/notification/notification.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from "react";
+import "./notification.css";
+
+function Notification({
+  notificationType,
+  notificationMessage,
+  showNotification,
+}) {
+  const [showAlert, setShowAlert] = useState(showNotification);
+  useEffect(() => {
+    if (showNotification === true) {
+      setShowAlert(true);
+      setTimeout(function () {
+        setShowAlert(false);
+      }, 5000);
+    }
+  }, [showNotification,notificationType,notificationMessage]);
```

Comment:
If you want to listen on every change you can use` [] `instead of `[showNotification, notificationType, notificationMessage])`